packages/account-v2/src/components/base/Dropzone/Dropzone.tsx

```diff
@@ -0,0 +1,165 @@
+import React, {
```

Comment:
Why are we putting it here instead of `deriv/ui`

Reply:
@aizad-deriv, P2P too have used Dropzone in their package. so, maybe i will just add a TODO comment in this PR for now. once we add the component to @deriv/ui(we will a create a card for that, and will work on it next), we will import it from there.
cc: @likhith-deriv @matin-deriv